.github/workflows/master-push.yml

```diff
@@ -148,12 +148,13 @@ jobs:
         run: |
           git checkout -B gh-pages
           cd web
+          git submodule init k-web-theme && git submodule update --recursive
```

Comment:
Hmmmm, why would this be needed? That submodule already exists in the repository. Shouldn't it be already included in the recursive submodule checkout above?

Also, I'm not even sure we're still using this submodule. @shd101wyy do we still need this submodule?

Reply:
I wass seeing it as part of the package.json definition for 'build install run'. 

The submodule you're right should be there. However when going through these steps. it's not being checked out when calling 'git submodule update --recursive' ... this could just be a local issue I was having each time I started with a fresh clone. I'll pull it for now and lets push this through with the gitmodules getting added back.